Resilio Connect API/JSMonitor/data-store.js

```diff
@@ -40,3 +43,21 @@ function getJobProperty(id, propertyName) {
         throw "No such id or propertyName";
     }    
 }
+
+function enumerateAgents() {
```

Comment:
This function should use what's already in the data store.   It should not make an API call.
The data-store should stay "pure"  - it is only about storing, getting, and manipulating data.